### DISCUSSION TITLE
Spell checking?

### DISCUSSION THREAD

**Discussion by Merovex** [tauri-apps/tauri#7482] [Q&A] [answered]
How would I enable the operating system's built-in spell checking? I am tinkering with a text-editing application and realized I don't have a way to use spell checking.

**Comment by FabianLars** [comment, marked as answer]
Which OS?
At least on Windows it should be enabled by default.
On macOS it should kinda work if you add `spellcheck="true"` to the element but there won't be any visual indicators but right clicking on misspelled words should still open a menu with suggestions. -> I didn't test it myself but a community member reported this on Discord. Also idk how we could fix the styling.
No idea about Linux right now.

**Reply by Merovex** [reply]
I should have been clear on the OS; apologies. MacOS (for now).

Thanks. That does trigger the "spellcheck" behavior. You are correct: it does not show the visual indicators. I was going to explore embedding Language Tool in my app (Java app spawns API server), and was still scratching my head on the visual indicators. I also need to sort out grammar checking.

I'll mark this as answered, but I'll add to this if I sort out the visual indicators.

(I should also have searched Discord.)

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/spellcheck

**Reply by jarb99** [reply]
Did you resolve the indicators? This one has me stumped too
